src/dialogs/image-cropper-dialog/image-cropper-dialog.ts

```diff
@@ -74,7 +74,7 @@ export class HaImagecropperDialog extends LitElement {
           round: Boolean(this._params?.options.round),
         })}"
       >
-        <img />
+        <img alt=""/>
```

Comment:
I can't think of a good alternate rendering for the image in an image cropper. Feel free to suggest.